app/jobs/send_inactive_account_email_job.rb

```diff
@@ -0,0 +1,11 @@
+class SendInactiveAccountEmailJob < ApplicationJob
+  queue_as :default
+
+  def perform_now
+    return if DisableExpensiveJobs.enabled?
+
+    Jobseeker.where("DATE(last_sign_in_at) = ?", 5.years.ago.to_date).each do |jobseeker|
```

Comment:
So this sends the email to jobseekers who last signed in 5 years ago? Shouldn't it be jobseekers who last signed in 2 weeks less than 5 years ago?